docs/overview/supported-operations.md

```diff
@@ -1,7 +1,143 @@
 # Supported Operations
 
-## Compiler
+As of 2020-06-26
+- TensorFlow commit e5023a1738cce7efcdf9d87863b85c80ab2f8c9e
```

Comment:
I would like to add a rough version information as well as a commit hash. For example 'Tensorflow nightly build after v2.2.0 release.' , etc.